### PR TITLE
Add namespaces when creating elements

### PR DESCRIPTION
**Pull request by Wuestengecko**
In order for the types of model elements to be correctly identifiable, the namespace has to be defined on the fragment's respective root element. This commit adds new methods for this purpose, and ensures that `GenericElement.__init__` calls them with the correct values.

### CODE REVIEW

**Review by ewuerger — CHANGES_REQUESTED** [review]
I tested your implementation on a completely new Capella model and found out that the comment wasn't there anymore. `model.save()` would always give me the `"Cannot find Capella version: No version comment found"` warning, so I figured that there is something wrong.

Still the warning is there once when saving the model b/c in the `MelodyLoader.get_model_info` the `comment()` xpath isn't finding the version comment anymore... even though it is in the tree. I looked in it while debugging. I'd suggest that you add another test case that looks for all content before (comment) underneath (children) and after (probably nothing?) the project root element. Also it should be explained that capellambse isn't automating the tedious enabling of requirements via the viewpoints manager in Capella.

File: capellambse/loader/core.py [hunk 1]
```diff
@@ -244,6 +244,35 @@ def idcache_reserve(self, new_id: str) -> None:
...
+        new_root = self.root.makeelement(
+            self.root.tag,
+            attrib=self.root.attrib,
+            nsmap={**self.root.nsmap, name: uri},
+        )
+        new_root.extend(self.root.getchildren())
+
+        siblings = self.root.itersiblings(preceding=True)
+        for i in reversed(list(siblings)):
+            self.root.addprevious(i)
```
[the remark this answers is not in the document]
Comment (Wuestengecko) [reply capellambse/loader/core.py:267]: Nice catch. Yes, that was supposed to go to the `new_root`.

**Comment by Wuestengecko** [comment]
The automatic viewpoint activation is not going to be part of this PR, as that can quickly grow too far out of scope (it involves the AFM file, which we currently don't use for anything yet). I've opened issue #245 to keep track of that.

**Comment by Wuestengecko** [comment]
@ewuerger

> Still the warning is there once when saving the model b/c in the `MelodyLoader.get_model_info` the `comment()` xpath isn't finding the version comment anymore.

I can't repro after fixing the `new_root.addprevious()`. Can you retest with the newest version I just pushed? There's a new test for that, so you can just re-run the suite.

**Review by ewuerger — APPROVED** [review]
LGTM.

**Review by ewuerger — APPROVED** [review]
LGTM.

**Pull request merged**